cfgov/v1/migrations/0136_migrate_imagetextgroups_to_infounitgroup.py

```diff
@@ -0,0 +1,153 @@
+from django.db import migrations
+
+from v1.util.migrations import get_stream_data, set_stream_data
+
+
```

Comment:
It would be extremely helpful both to review this PR and for future developers looking at this migration to include a comment here documenting the different formats of these two things. For example, something like:

```py
"""
ImageText2575Group organisms has this format:

{
    # string, not required
    'heading': 'Heading',

    # boolean, not required
    'link_image_and_heading': True,

    # list of ImageText2575 molecules
    'image_texts': [
        {
            # string, not required
            'heading': 'Heading',

            # rich text, not required
            'body': 'Rich text, maybe with <a linktype="page" id="123">links</a>',

            # etc....
        },
        ...
    ]
}
"""
```

I know this information exists in the source files but having it in the migration would make it easier to understand what this is doing. Would adding something like this to both of these files be too much to ask?

Reply:
I'll see if I can do this without too much pain :)